truncate_client/src/web_comms.rs

```diff
@@ -0,0 +1,56 @@
+use eframe::web_sys;
+use futures::channel::mpsc::{Receiver, Sender};
+use futures::SinkExt;
+use futures_util::{future, pin_mut, StreamExt};
+use truncate_core::messages::{GameMessage, PlayerMessage};
+use ws_stream_wasm::{WsMessage, WsMeta};
+
+pub async fn connect(
+    connect_addr: String,
+    tx_game: Sender<GameMessage>,
+    rx_player: Receiver<PlayerMessage>,
+) {
+    use web_sys::console;
+
+    console::log_1(&format!("Connecting to {connect_addr}").into());
```

Comment:
Sure does, again the web_sys with some browser APIs plugs that all together. (`log_1` because Rust functions don't have variadic args, `log_2` takes _two_ strings etc)